### ISSUE TITLE
Remove `rusk-wallet` from this repository

### ISSUE THREAD

**Issue by ZER0** [dusk-network/rusk#687]
**Describe what you want implemented** 
Remove `rusk-wallet` from this repository 

**Describe "Why" this is needed**
Having the wallet as part of the workspace makes working on it slightly harder than if it was a separate repo. Also, we want to test the wallet in different OS such as Windows, that are not necessary a pre-requisite for Rusk itself, at least not now, therefore the CI can't run on our self host machine.

**Describe alternatives you've considered**
Keep as is.

**Additional context**
The wallet depends on rusk's protobuf schema; in order to have the wallet compiled as separate repo, #686 needs to be fixed first. However this issue is only about remove the wallet from this repository, so it's not blocked by #686.
